vre/values.yaml

```diff
@@ -0,0 +1,485 @@
+        #     claimName: cvmfs-vre-pvc
+        # - name: eospilot-eulake # mounts the EOS RSE needed for the Rucio JupiterLab extension
+        #   hostPath:
+        #     # path: /var/eos/eulake # This is pointing to /eos/pilot/eulake/escape/data, defined on the configmap/eos-csi-dir-etc-eo  
+        #     path: /var/eos/pilot
+        # - name: eos-atlas
+        #   hostPath:
+        #     path: /var/eos/atlas
+      extraVolumeMounts: []
+        # - name: cvmfs-vre
+        #   mountPath: /cvmfs
+        #   # CVMFS automount volumes must be mounted with HostToContainer mount propagation.
+        #   mountPropagation: HostToContainer
+        # - name: eospilot-eulake # mounts the EOS RSE needed for the Rucio JupiterLab extension
+        #   mountPath: /eos/eulake
+        #   mountPropagation: HostToContainer
+        #   readOnly: true 
+        # - name: eos-atlas
+        #   mountPath: /eos/atlas
+        #   mountPropagation: HostToContainer
+        #   readOnly: true
+    image:
+      name: ghcr.io/vre-hub/vre-singleuser-py311
+      tag: sha-281055c
+      pullPolicy: Always
+
+
+    profileList:
+      - display_name: "Default environment"
+        description: "Based on a scipy notebook environment with a python-3.11 kernel, the rucio jupyterlab extension and the reana client installed."
+        default: true
+
+    # nodeSelector: 
+    #   jupyter: singleuser
+    # extraTolerations:
+    #   - key: jupyter
+    #     operator: Equal
+    #     value: singleuser
+    #     effect: NoSchedule
+    # memory:
+    #   limit: 3.5G #4G
+    #   guarantee: 3G #2G
+
+    cmd: null
+    extraEnv:
+  #     JUPYTERHUB_SINGLEUSER_APP: "notebook.notebookapp.NotebookApp"
+      RUCIO_MODE: "replica"
+      RUCIO_WILDCARD_ENABLED: "1"
+      RUCIO_BASE_URL: "https://vre-rucio.cern.ch"
+      RUCIO_AUTH_URL: "https://vre-rucio-auth.cern.ch"
+      RUCIO_WEBUI_URL: "https://vre-rucio-ui.cern.ch"
+      RUCIO_DISPLAY_NAME: "RUCIO - CERN VRE"
+      RUCIO_NAME: "vre-rucio.cern.ch"
+      RUCIO_SITE_NAME: "CERN"
+      RUCIO_OIDC_AUTH: "env"
+      RUCIO_OIDC_ENV_NAME: "RUCIO_ACCESS_TOKEN"
+      RUCIO_DEFAULT_AUTH_TYPE: "oidc"
+      RUCIO_OAUTH_ID: "rucio"
+      RUCIO_DEFAULT_INSTANCE: "vre-rucio.cern.ch"
+      RUCIO_DESTINATION_RSE: "CERN-EOSPILOT"
+      RUCIO_RSE_MOUNT_PATH: "/eos/eulake"
+      RUCIO_PATH_BEGINS_AT: "5" # because we are need to substitute /eos/pilot/eulake/escape/data with /eos/eulake
+      RUCIO_CA_CERT: "/certs/rucio_ca.pem"
+      OAUTH2_TOKEN: "FILE:/tmp/eos_oauth.token"
+
+    # extraFiles:
+    #   jupyter_notebook_config.json:
+    #     mountPath: /etc/jupyter/jupyter_notebook_config.json
+    #     data:
+    #       MappingKMappingKernelManager:
+    #         cull_idle_timeout: 3600
+    #         cull_interval: 600
+    #         cull_busy: false
+    #         cull_connected: true
+
+  prePuller:
+    hook:
+      enabled: true
+
+  ingress:
+    enabled: true
+    ingressClassName: null
```

Comment:
This is at least confusing to me, the annotations below assume traefik

Reply:
there is a traefik built in reana